docs/make.jl

```diff
@@ -26,21 +27,23 @@ end
 makedocs(;
     modules=[AbstractDifferentiation],
     authors="Mohamed Tarek <mohamed82008@gmail.com> and contributors",
-    repo="https://github.com/JuliaDiff/AbstractDifferentiation.jl/blob/{commit}{path}#{line}",
     sitename="AbstractDifferentiation.jl",
     format=Documenter.HTML(;
+    repolink="https://github.com/JuliaDiff/AbstractDifferentiation.jl",
         prettyurls=get(ENV, "CI", "false") == "true",
         canonical="https://JuliaDiff.github.io/AbstractDifferentiation.jl",
         edit_link="master",
         assets=String[],
     ),
     pages=[
         "Home" => "index.md",
-        "API reference" => "api.md",
+        "User guide" => "user_guide.md",
+        "Implementer guide" => "implementer_guide.md",
     ],
 )
 
 deploydocs(;
     repo="github.com/JuliaDiff/AbstractDifferentiation.jl",
     devbranch="master",
+    push_preview=true,
```

Comment:
Could you add the Github action that removes these previews automatically?

Reply:
do you have a specific one in mind?

Reply:
There's one in the Documenter docs that should work I assume (I guess over time possibly I accumulated smaller modifications in my repos but they should not matter here).

Reply:
https://documenter.juliadocs.org/stable/man/hosting/#gh-pages-Branch